openfisca_core/variables.py

```diff
@@ -27,36 +27,42 @@
     bool: {
         'dtype': np.bool,
         'default': False,
-        'json_type': 'Boolean',
+        'json_type': 'boolean',
+        'formatted_value_type': 'Boolean',
```

Comment:
> would `python_type` make more sense ? I explains why we display this in the legexp.

Reply:
I'd expect something called `python_type` to be `'float'`, `'int'` or `'bool'`.